cli/src/checks.rs

```diff
@@ -93,7 +93,7 @@ pub fn check_account_for_balance_with_commitment(
     let lamports = rpc_client
         .get_balance_with_commitment(account_pubkey, commitment)?
         .value;
-    if lamports != 0 && lamports >= balance {
+    if lamports >= balance {
```

Comment:
I think we want to keep this?  https://github.com/solana-labs/solana/pull/8204

Reply:
Ah, red-herring fix. Good catch.